Modules/System/Time Zone Management/src/TimeZoneInfoInitializer.Codeunit.al

```diff
@@ -0,0 +1,23 @@
+codeunit 8724 "TimeZoneInfo Initializer"
```

Comment:
We usually don't CamelCase object names.

Reply:
I did that in this case, since the DotNet object being initialized is TimeZoneInfo, but I have added spaces.